contracts/chain-adapters/Optimism_Adapter.sol

```diff
@@ -37,7 +41,7 @@ contract Optimism_Adapter is CrossDomainEnabled, AdapterInterface {
     address public immutable dai = 0x6B175474E89094C44Da98b954EedeAC495271d0F;
     address public immutable daiOptimismBridge = 0x10E6593CDda8c58a1d0f14C5164B376352a55f2F;
     address public immutable snx = 0xC011a73ee8576Fb46F5E1c5751cA3B9Fe0af2a6F;
-    address public immutable snxOptimismBridge = 0xCd9D4988C0AE61887B075bA77f08cbFAd2b65068;
+    address public immutable snxOptimismBridge = 0x39Ea01a0298C315d149a490E34B59Dbf2EC7e48F;
```

Comment:
Was our previous code just wrong?

Reply:
Yeah I got this new address from [here](https://github.com/ethereum-optimism/ethereum-optimism.github.io/blob/master/data/SNX/data.json) but it does look like 0xCd is the same contract that hasn't been used in 280 days... so I'm a bit confused how I got it

Reply:
Here's the original PR https://github.com/across-protocol/contracts-v2/pull/68